samples/wrapsizer/wrapsizer.cpp

```diff
@@ -118,10 +118,13 @@ WrapSizerFrame::WrapSizerFrame()
 
         sizerMidWrap->Add(chk, wxSizerFlags().Centre().Border());
     }
-
     sizerMid->Add(sizerMidWrap, wxSizerFlags(100).Expand());
     sizerRoot->Add(sizerMid, wxSizerFlags(100).Expand().Border());
 
+    // A long wxStaticText that wraps like a wxWrapSizer
```

Comment:
Really not sure this belongs to this sample anyhow. I'd rather add a button setting a long label to the "widgets" sample.